lib/workos/user_management.rb

```diff
@@ -275,6 +275,33 @@ def verify_email(magic_auth_challenge_id:, code:)
 
         WorkOS::User.new(response.body)
       end
+
+      # Updates user user password.
+      #
+      # @param [String] id The unique ID of the User.
+      # @param [String] new_password The new password to set for the user.
```

Comment:
Looks like this should be `password`

```suggestion
      # @param [String] password The new password to set for the user.
```